torchdynamo/guards.py

```diff
@@ -97,7 +97,18 @@ def __lt__(self, other):
         return self.sort_key() < other.sort_key()
 
     def __str__(self):
-        return f"{self.source.name.lower()} {repr(self.name)} {self.create_fn.__name__} EXPORT_INFO: {{'guard_type': {self.guard_type}, 'code': {self.code}, 'obj_id': {self.obj_id}, 'guarded_class': {self.guarded_class}}}"
+        s = f"{self.source.name.lower()} {repr(self.name)} {self.create_fn.__name__}"
```

Comment:
i'm not up to speed on this but why is it only possible to print the details such as self.guard_type under export?  It looks like it was previously possible all the time, unless you are now just preventing trying to print when printing is not possible

Reply:
No, this has nothing to do with possible/impossible printing, but rather with keeping things as they were.

Before this pr, the print statement was:

```
f"{self.source.name.lower()} {repr(self.name)} {self.create_fn.__name__}"
```

When `export_guards` is disabled, I'd like to stay exactly as it is before this PR. If we were to not have this config check, this info would be null as the other parts of the PR responsible for setting this data will not have been called, so it would be useless anyway.